src/_locales/zh_CN/messages.yml

```diff
@@ -418,6 +418,9 @@ labelNotifyUpdates:
 labelNotifyUpdatesGlobal:
   description: Option to prioritize global notification option over script's setting.
   message: 忽略每个脚本的提示（编辑器中的“设置”页）
+labelUpdatesParallel:
+  description: 当多个脚本的更新地址为同一个域时,会根据设置的并发数进行分批次更新
```

Comment:
`,` -> `，`

Reply:
We only use full-width punctuation for Chinese, please don't change those in English.

Reply:
OK